app/classifier/feedback/feedback-summary-container.jsx

```diff
@@ -35,24 +38,30 @@ class FeedbackSummaryContainer extends React.Component {
     }, []);
     actions.feedback.setFeedback(feedbackItems);
   }
+
+  render() {
+    return (this.props.feedback.length) ? <FeedbackSummary feedback={this.props.feedback} /> : null;
```

Comment:
`this.props.feedback.length` always returns `true`; so I think it should be `this.props.feedback.length > 0`.